todo_app/templates/index.html

```diff
@@ -9,10 +9,34 @@ <h1 class="display-4">To-Do App</h1>
 
   <div class="row">
     <div class="col-md-12">
+      <form method="post" action="/items">
+        <input name="new_item" type="text" />
```

Comment:
Really minor, but sticking a `required` attribute here to ensure I can't accidentally submit an empty form would be nice